extensions/import/src/wizard/pages/modifyColumnsPage.ts

```diff
@@ -63,30 +66,49 @@ export class ModifyColumnsPage extends ImportPage {
 		return [column.columnName, column.dataType, false, column.nullable];
 	}
 
+	private static convertRow(row: any[]): ColumnMetadata {
```

Comment:
Just have this be a function outside the class, no reason to have it be a member of this class